src/javascript/app_2/Modules/Trading/Containers/trade-params.jsx

```diff
@@ -47,10 +130,61 @@ TradeParams.propTypes = {
+        ui_store              : ui,
+        // Amount component props
+        amount                : modules.trade.amount,
+        basis                 : modules.trade.basis,
+        basis_list            : modules.trade.basis_list,
+        contract_start_type   : modules.trade.contract_start_type,
+        contract_type         : modules.trade.contract_type,
+        contract_types_list   : modules.trade.contract_types_list,
+        currencies_list       : client.currencies_list,
+        currency              : modules.trade.currency,
+        duration_unit         : modules.trade.duration_unit,
+        expiry_type           : modules.trade.expiry_type,
+        is_equal              : modules.trade.is_equal,
+        // is_minimized          : modules.trade.,
+        // is_nativepicker       :,
+        is_single_currency    : client.is_single_currency,
+        onChange              : modules.trade.onChange,
+        validation_errors     : modules.trade.validation_errors,
+        // Barrier component props
+        barrier_1             : modules.trade.barrier_1,
+        barrier_2             : modules.trade.barrier_2,
+        barrier_count         : modules.trade.barrier_count,
+        // is_minimized          :,
+        // onChange              : modules.trade.onChange,
+        // validation_errors     : modules.trade.validation_errors,
+        // Duration component props
+        advanced_duration_unit: ui.advanced_duration_unit,
+        advanced_expiry_type  : ui.advanced_expiry_type,
+        contract_expiry_type  : modules.trade.contract_expiry_type,
+        duration              : modules.trade.duration,
+        // duration_unit         : modules.trade.duration_unit,
+        duration_units_list   : modules.trade.duration_units_list,
+        duration_min_max      : modules.trade.duration_min_max,
+        duration_t            : ui.duration_t,
+        expiry_date           : modules.trade.expiry_date,
+        expiry_time           : modules.trade.expiry_time,
+        // expiry_type           : modules.trade.expiry_type,
+        getDurationFromUnit   : ui.getDurationFromUnit,
+        // hasDurationUnit       : ,
+        is_advanced_duration  : ui.is_advanced_duration,
+        // is_minimized          :,
+        // onChange              : modules.trade.onChange,
+        onChangeUiStore       : ui.onChangeUiStore,
+        onChangeMultiple      : modules.trade.onChangeMultiple,
+        simple_duration_unit  : ui.simple_duration_unit,
+        server_time           : common.server_time,
+        start_date            : modules.trade.start_date,
+        // validation_errors     : modules.trade.validation_errors,
+        market_open_times     : modules.trade.market_open_times,
+        // LastDigit component props
+        // is_minimized          :,
+        last_digit            : modules.trade.last_digit,
+        // onChange              : modules.trade.onChange,
```

Comment:
What's the reason for passing down all the mobx props through the `TradeParams` component? Can we connect the child components inside Duration, Barrier, LastDigit, Amount instead?

Reply:
@OskarAhl That is exactly the plan but the change will come in the next PR.